br/pkg/utils/safe_point.go

```diff
@@ -21,8 +21,9 @@ const (
 	brServiceSafePointIDFormat      = "br-%s"
 	preUpdateServiceSafePointFactor = 3
 	checkGCSafePointGapTime         = 5 * time.Second
-	// DefaultBRGCSafePointTTL means PD keep safePoint limit at least 5min.
-	DefaultBRGCSafePointTTL = 5 * 60
+	// DefaultBRGCSafePointTTL means PD keep safePoint limit at least 72h.
+	DefaultBRGCSafePointTTL = 72 * 60 * 60 // 72h
```

Comment:
Defaults to 72 hours may be too long, and waste disk space if there is no following CDC.
Maybe we can keep 5 minutes, but tell CDC users to set a suitable value.

Reply:
done